ext/sodium/libsodium.c

```diff
@@ -3156,23 +3037,23 @@ PHP_FUNCTION(sodium_crypto_kdf_derive_from_key)
 		RETURN_THROWS();
 	}
 	if (subkey_len < crypto_kdf_BYTES_MIN) {
-		zend_throw_exception(sodium_exception_ce, "subkey cannot be smaller than SODIUM_CRYPTO_KDF_BYTES_MIN", 0);
+		zend_argument_error(sodium_exception_ce, 1, "to be more than or equal to SODIUM_CRYPTO_KDF_BYTES_MIN");
```

Comment:
more than -> greater than

Reply:
Also, it's possible to formulate this as `to be at least BYTES_MIN` and `to be at most BYTES_MAX`, which would be more compact.

Reply:
So do you think we should use `at least` and `at most` instead of `greater than or equal to` and `less than or equal to` in general? I'll update my PRs then

Reply:
I think so. It's more concise and, at least to me, just as clear. I could see an argument being made that "greater than or equal" is more obvious though.

Reply:
Weell, it does depend on the case. "must be at least 10 bytes long" is definitely better than "must be greater than or equal to 10 bytes long", but I'm not so sure about "must be at least 10" and "must be greater than or equal to 10". The case of "most be at least 0" specifically sounds outright weird (but can be "must be non-negative" or "must not be negative").

So overall: Not sure :)

Reply:
@Girgias Do you maybe have any other suggestion or preference? On the one hand, I like the `greater than or equal to` form because it's obvious and it's the most consistent with `greater than`. On the other hand, `at least` is surely a more concise way to say the same.  Additionally, we have `positive`/`negative`/`non-negative` etc. which - at least for me - need a little bit more processing time. Furthermore, according to what I remember, they are not ubiquitous everywhere (see my comment more below. Or it is just a myth?).

That said, my preference is to use `greater/less than or equal to` for pure minimum/maximum values, and `at least/at most` for minimum/maximum length etc.. It's not a strong opinion though, so I welcome any opposition. :)

Reply:
I would prefer the explicit ``greater/less than or equal to`` and ``at least/at most`` as it is straight forward, positivity and negativity is where French and English doesn't really agree on, famous example is how ℕ is ambiguous as French mathematician will include 0 in it whereas British won't for the most part (as it's meant to represent positive numbers).
This is mainly due to how in French "greater" and "less" are inclusive.

It also seems Japanese has this ambiguity (they literally have a brand called +-0 https://www.plusminuszero.jp/)